openquake/hazardlib/gsim/stewart_2016_vh.py

```diff
@@ -0,0 +1,204 @@
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`StewartEtAl2016VH`,
+               :class:`StewartEtAl2016RegCHNVH`,
+               :class:`StewartEtAl2016RegJPNVH`,
+               :class:`StewartEtAl2016NoSOFVH`,
+               :class:`StewartEtAl2016RegCHNNoSOFVH`,
+               :class:`StewartEtAl2016RegJPNNoSOFVH`,
+"""
+
+import numpy as np
+
+import openquake.hazardlib.gsim.boore_2014 as BSSA
+import openquake.hazardlib.gsim.bozorgnia_campbell_2016_vh as BC15b
+from openquake.hazardlib.gsim.base import GMPE
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV, SA
+
+def _get_stddevs(cls, C, sites, rup, dists, imt, stddev_types):
+    """
+    Returns the inter-event, intra-event, and total standard deviations
+    """
+    C_V = cls.VGMPE.COEFFS[imt]
+    C_H = cls.HGMPE.COEFFS[imt]
+    inter_std = [const.StdDev.INTER_EVENT]
+    intra_std = [const.StdDev.INTRA_EVENT]
+    num_sites = len(sites.vs30)
+    
+    tau_v = BSSA._get_stddevs("stewart", C_V, rup, dists, sites, inter_std)
+    tau_h = BSSA._get_stddevs("base", C_H, rup, dists, sites, inter_std)
+    phi_v = BSSA._get_stddevs("stewart", C_V, rup, dists, sites, intra_std)
+    phi_h = BSSA._get_stddevs("base", C_H, rup, dists, sites, intra_std)
+    tau = _get_tau_vh(C, rup.mag, tau_v, tau_h)
+    phi = _get_phi_vh(C, rup.mag, phi_v, phi_h)
+    
+    stddevs = []
+    for stddev_type in stddev_types:
+        if stddev_type == const.StdDev.TOTAL:      
+            stddevs.append(np.sqrt((tau ** 2.) + (phi ** 2.)) +
+                            np.zeros(num_sites))
+        elif stddev_type == const.StdDev.INTRA_EVENT:
+            stddevs.append(phi + np.zeros(num_sites))
+        elif stddev_type == const.StdDev.INTER_EVENT:
+            stddevs.append(tau + np.zeros(num_sites))
+    # return std dev values for each stddev type in site collection
+    return stddevs
+
+def _get_tau_vh(C, mag, stddev_v, stddev_h):
+    """
+    Returns the inter-event random effects coefficient (tau) defined in
+    Equation 13c.
+    """
+    return BC15b._get_tau_vh(C, mag, stddev_v, stddev_h)
+
+def _get_phi_vh(C, mag, stddev_v, stddev_h):
+    """
+    Returns the intra-event random effects coefficient (phi) defined in
+    Equation 13b.
+    """
+    return BC15b._get_phi_vh(C, mag, stddev_v, stddev_h)
+
```

Comment:
Can't you just call  BC15b._get_tau_vh and  BC15b._get_phi_vh?

Reply:
Yes that would be possible. Should the docstring be a comment instead or just removed altogether?